src/core_modules/capture-core/components/Pages/New/RegistrationDataEntry/RegistrationDataEntry.epics.js

```diff
@@ -124,18 +126,20 @@ export const completeSavingNewTrackedEntityInstanceEpic: Epic = (action$: InputO
 export const startSavingNewTrackedEntityInstanceWithEnrollmentEpic: Epic = (action$: InputObservable, store: ReduxStore) =>
     action$.pipe(
         ofType(registrationFormActionTypes.NEW_TRACKED_ENTITY_INSTANCE_WITH_ENROLLMENT_SAVE_START),
-        map(() => {
+        map((action) => {
             const { currentSelections: { orgUnitId, programId }, formsValues, dataEntriesFieldsValue } = store.value;
             const { incidentDate, enrollmentDate, geometry } = dataEntriesFieldsValue['newPageDataEntryId-newEnrollment'] || { };
```

Comment:
Not really a part of this issue, but it would be great to fix it anyway. `incidentDate` and `enrollmentDate` should also be converted (formToClient then clientToServer). The reason is that we can display the date in the format DD-MM-YYYY (we have a setting for this), but the format sent to the server should always be YYYY-MM-DD. Doing the conversions should fix the potential problem here.

Reply:
Hey, 
Yes, it makes sense to fix `incidentDate` and `enrollmentDate`. I updated the code with your suggestions. 
Thank you!